sound/soc/sof/loader.c

```diff
@@ -86,7 +86,8 @@ static int get_cc_info(struct snd_sof_dev *sdev,
 }
 
 /* parse the extended FW boot data structures from FW boot message */
-int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 bar, u32 offset)
+int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, unsigned int bar,
```

Comment:
bar is a 32-bit address, no? Why would we remove that information

Reply:
no, it's an array index.

Reply:
ack on that, it's "sdev->bar[bar] + offset", so this seems good